x/gamm/types/expected_keepers.go

```diff
@@ -45,3 +47,7 @@ type CommunityPoolKeeper interface {
 type PoolIncentivesKeeper interface {
 	IsPoolIncentivized(ctx sdk.Context, poolId uint64) bool
 }
+
+type PoolCreationManager interface {
```

Comment:
We should name this as `SwaprouterKeeper` as swap logic is going to be defined on top of this later.